tests/helpers/test_groq_stream.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import sys
+import os
+import unittest
+from groq import AsyncGroq
+from dotenv import dotenv_values
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../..")))
```

Comment:
Does this work with context.py?
```suggestion
from .context import iftk
```

Reply:
It might have something to do with relative imports and how `unittest` handles running multiple test files, but it doesn't seem to work even with other workarounds.

Reply:
OK, we can leave this as-is and figure it out later.